src/Toggle/Toggle.js

```diff
@@ -204,7 +209,7 @@ class Toggle extends React.Component {
     );
 
     if (this.state.switched) {
-      thumbStyles.marginLeft = `-${thumbStyles.width}`;
+      thumbStyles.marginLeft = 0 - thumbStyles.width;
```

Comment:
Sorry for the late reply. But things will break right because of this inconsistency. 
Hope this will help
`0 - parseInt(`${thumbStyles.width}`.replace('px', ''), 10)`